lib/ecosystems/enrich_spdx.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/rs/zerolog"
 	"github.com/spdx/tools-golang/spdx"
 	"github.com/spdx/tools-golang/spdx/v2/v2_3"
+	"github.com/spdx/tools-golang/spdx/v2/common"
```

Comment:
This has to move one line above (bfore the v2.3 import).

Reply:
I just added a (signed) commit to move the import with should resolve this.